File: Naloga 19/naloga_19.py
def naloga_19():
    def deljivost(x, y):
        if x % y == 0:
            return True
    for stevilo in range(1, 500):
        if deljivost(stevilo, 4):
            if not deljivost(stevilo, 10):
                print(stevilo)

if __name__ == '__main__':
    naloga_19()

File: Naloga 19/naloga_19.cpp
#include <iostream>

bool deljivost(int x, int y) {
    if ((x % y) == 0) {
        return true;
    }
    return false;
}
void naloga_19() {
    for (int stevilo = 1; stevilo<=500; stevilo++) {
        if (deljivost(stevilo, 4)) {
            if (!(deljivost(stevilo, 10))) {
                std::cout << stevilo << std::endl;
            };
        };
        
    };
};

int main() {
    naloga_19();
    return 0;
};